Tests/SwifterSwiftTests/UIKitExtensionsTests/UISliderExtensionsTests.swift

```diff
@@ -13,6 +13,18 @@ import XCTest
 
 class UISliderExtensionsTests: XCTestCase {
 
+    func testCompletionCalledAnimated() {
+        let slider = UISlider()
+        slider.minimumValue = 0
+        slider.maximumValue = 100
+        let exp = expectation(description: "calledCompletion")
+        slider.setValue(90, animated: true, duration: 0.5) {
+            XCTAssertEqual(slider.value, 90.0)
+            exp.fulfill()
+        }
+        waitForExpectations(timeout: 1, handler: nil)
```

Comment:
Can we just use a timeout of 5 for all of these? I just want to make sure it's enough.